pkg/systemd/overview.jsx

```diff
@@ -38,8 +39,116 @@ import { MotdCard } from './overview-cards/motdCard.jsx';
+                            obj.type = "info";
+                        }
+
+                        this.setState({ messages: obj });
+                    }
+                })
+                .fail(function (error) {
+                    console.error("failed to fetch login messages: ", error);
+                });
+    }
+
+    close() {
+        const bridge = cockpit.dbus(null, { bus: "internal" });
+        bridge.call("/LoginMessages", "cockpit.LoginMessages", "Dismiss", [])
+                .fail(function (error) {
+                    console.error("failed to dismiss login messages: ", error);
+                });
+
+        this.setState({ messages: {} });
+    }
+
+    goToLogs() {
+        if (this.permission && this.permission.user)
+            cockpit.jump("/users#/" + this.permission.user.name);
+    }
+
+    render() {
+        const messages = this.state.messages;
+
+        // Do the full combinatorial thing to improve translatability
+        function generate_line(host, line, datetime) {
+            let message = "";
+            if (host && line) {
+                message = cockpit.format(_("<date> from <host> on <terminal>", "$0 from $1 on $2"), datetime, host, line);
+            } else if (host) {
+                message = cockpit.format(_("<date> from <host>", "$0 from $1"), datetime, host);
+            } else if (line) {
+                message = cockpit.format(_("<date> on <terminal>", "$0 on $1"), datetime, line);
+            } else {
+                message = datetime;
+            }
+            return message;
+        }
+
+        let last_login_message;
+        if (messages['last-login-time']) {
+            const datetime = moment.unix(messages['last-login-time']).format('ll LTS');
+            const host = messages['last-login-host'];
+            const line = messages['last-login-line'];
+            last_login_message = generate_line(host, line, datetime);
+        }
+
+        let last_fail_message;
+        if (messages['last-fail-time']) {
+            const datetime = moment.unix(messages['last-fail-time']).format('ll LTS');
+            const host = messages['last-fail-host'];
+            const line = messages['last-fail-line'];
+            last_fail_message = generate_line(host, line, datetime);
+        }
+
+        let fail_count_message = null;
+        if (messages['fail-count']) {
+            fail_count_message = cockpit.format(cockpit.ngettext("There was $0 failed login attempt since the last successful login.", "There were $0 failed login attempts since the last successful login.", messages['fail-count']), messages['fail-count']);
+        }
+
+        if (!last_login_message && !fail_count_message && !last_fail_message)
+            return null;
+
+        const last_log_item = <p><b>{_("Last login:")}</b> {last_login_message}</p>;
+        const last_fail_item = <p><b>{_("Last failed login:")}</b> {last_fail_message} <a onClick={this.goToLogs}>Account logs</a></p>;
```

Comment:
"Accout logs" must be i18n'ed. Should be enough to add a `translate` attribute to the `<a>`.

Reply:
Please comment these `Account logs` out and point it to https://github.com/cockpit-project/cockpit/issues/28 which actually requests exactly that. Interesting fact: It is our oldest issue we have :)